packages/application-generic/src/usecases/trigger-broadcast/trigger-broadcast.usecase.ts

```diff
@@ -30,7 +30,7 @@ import { SubscriberProcessQueueService } from '../../services/queues/subscriber-
 import { TriggerBroadcastCommand } from './trigger-broadcast.command';
 
 const LOG_CONTEXT = 'TriggerBroadcastUseCase';
-const QUEUE_CHUNK_SIZE = 100;
+const QUEUE_CHUNK_SIZE = process.env.QUEUE_CHUNK_SIZE ?? 100;
```

Comment:
I wonder should we maybe have 2 separate more specific names here?

```suggestion
const QUEUE_CHUNK_SIZE = process.env.BROADCAST_QUEUE_CHUNK_SIZE ?? 100;
```

and for the multicast `MULTICAST_QUEUE_CHUNK_SIZE` wdyt?

Reply:
I agree! Will apply your suggestion

Reply:
I also forgot to parse int from process env string. Will add parseInt as well